.dockerignore

```diff
@@ -1 +1,18 @@
-node_modules
\ No newline at end of file
+# local build artifacts
```

Comment:
Generic question what does the dockerignore do for us?  I guess I'm asking why we need to ignore these local build artifacts?

Reply:
`dockerignore` prevents local files from being thrown into the image. You can see from the log files of the failed deploy that we could stand to add a few more exclusions (OCP doesn't need git logs to run the app): 

![image](https://user-images.githubusercontent.com/2048170/141189133-c96d7ea9-9bb5-44d1-8b33-86e33a91427b.png)

The directories 'build', 'tmp' and 'node_modules' are local artifacts, that are created as part of the image build during `RUN` instructions in the Dockerfile. Not unlike local IDE and OS cruft they are artifacts that can also be created locally and thrown into a locally built image.